packages/eslint-plugin-pf-codemods/src/rules/v6/mastheadStructureChanges/masthead-structure-changes.test.ts

```diff
@@ -0,0 +1,165 @@
+const ruleTester = require("../../ruletester");
+import * as rule from "./masthead-structure-changes";
+
+ruleTester.run("masthead-structure-changes", rule, {
+  valid: [
+    {
+      code: `<Masthead  />`,
+    },
+    {
+      code: `import { Masthead } from '@patternfly/react-core'; <Masthead someOtherProp />`,
+    },
+  ],
+  invalid: [
+    // stage one of a pre-renamed file
+    {
+      code: `import { Masthead, MastheadBrand, MastheadMain, MastheadToggle } from '@patternfly/react-core'; <Masthead><MastheadToggle>Foo</MastheadToggle><MastheadMain><MastheadBrand>Bar</MastheadBrand></MastheadMain></Masthead>`,
+      output: `import { Masthead, MastheadBrand, MastheadMain, MastheadToggle } from '@patternfly/react-core'; <Masthead><MastheadMain><MastheadToggle>Foo</MastheadToggle><MastheadBrand>Bar</MastheadBrand></MastheadMain></Masthead>`,
+      errors: [
+        {
+          message: `The structure of Masthead has been updated, MastheadToggle and MastheadBrand should now be wrapped in MastheadMain.`,
```

Comment:
Nit not a blocker: wdyt about updating the message that gets output to only mention the applicable element? Instead of saying both "toggle and brand...", on a MastheadToggle, only mention "toggle...".